src/core/logging/index.ts

```diff
@@ -0,0 +1,20 @@
+/* eslint-disable @typescript-eslint/no-explicit-any */
+import type { Logger } from 'hyperview/src/types';
+
+let logger: Logger = console;
+
+function initialize(loggerInstance: Logger | undefined): void {
+  if (loggerInstance) {
+    logger = loggerInstance;
+  }
+}
+
+export default {
+  initialize,
+  log: (m?: any, ...p: any[]) => logger.log(m, p),
+  // eslint-disable-next-line sort-keys
+  info: (m?: any, ...p: any[]) => logger.info(m, p),
+  warn: (m?: any, ...p: any[]) => logger.warn(m, p),
+  // eslint-disable-next-line sort-keys
+  error: (m?: any, ...p: any[]) => logger.error(m, p),
+};
```

Comment:
I think you might be able to instead export these two and get rid of the default export, which also removes the need for disabling the "any" TS rule:

```suggestion
let logger: Logger = console;

export default logger;

export function initialize(loggerInstance: Logger | undefined): void {
  if (loggerInstance) {
    logger = loggerInstance;
  }
}
```

Reply:
Had to do a slightly different approach